docs/docsite/rst/roadmap/COLLECTIONS_5.rst

```diff
@@ -0,0 +1,56 @@
+.. _ansible_4_roadmap:
+
+===================
+Ansible project 5.0
+===================
+
+This release schedule includes dates for the `ansible <https://pypi.org/project/ansible/>`_ package, with a few dates for the `ansible-core <https://pypi.org/project/ansible-core/>`_ package as well. All dates are subject to change. See :ref:`base_roadmap_2_12` for the most recent updates on ``ansible-core``.
+
+.. contents::
+   :local:
+
+
+Release schedule
+=================
+
+TBD
```

Comment:
```suggestion
```

Reply:
(It was approved at last week's community meeting)

Reply:
Heh, I put that in when I wasn't going to fill in all the details, then forgot about it. Thanks for bringing it back to my attention @dmsimard.